.travis.yml

```diff
@@ -12,7 +12,17 @@ env:
   - TOXENV=py33-1.8
   - TOXENV=py34-1.8
   - TOXENV=py34-1.9
+  - TOXENV=py34-1.10
+  - TOXENV=py34-1.11
   - TOXENV=py34-master
+  - TOXENV=py35-1.9
+  - TOXENV=py35-1.10
+  - TOXENV=py35-1.11
```

Comment:
I don't get it, because this isn't required (nor documented on that travis help page), for example this project works just fine for all pythons (non-django project but still); https://github.com/rm-hull/luma.core/blob/master/.travis.yml